src/commands.cc

```diff
@@ -2556,19 +2557,34 @@ const CommandDesc declare_user_mode_cmd = {
     "declare-user-mode",
     nullptr,
     "declare-user-mode <name>: add a new user keymap mode",
-    single_param,
+    ParameterDesc{
+        { { "docstring", { true, "user mode description" } } },
+        ParameterDesc::Flags::SwitchesOnlyAtStart, 1, 2
```

Comment:
Any reason for forcing switches at the start ? We usually do that only for commands that might get other commands/things that look like switch later on, here it seems like an unnecessary constraint.

Reply:
All commands show switches as being the first optional list of flags, so I guess I used this flag here.

I reverted back to `None`.